pkg/tnf/handlers/podsets/podsets_test.go

```diff
@@ -78,32 +78,33 @@ func Test_ReelMatchSuccess(t *testing.T) {
 
 // Just ensure there are no panics.
 func Test_ReelEof(t *testing.T) {
-	newDp := dp.NewDeployments(testTimeoutDuration, testNamespace)
+	newDp := ps.NewPodSets(testTimeoutDuration, testNamespace, resourceType)
 	assert.NotNil(t, newDp)
 	newDp.ReelEOF()
 }
 
 const (
+	resourceType             = "deployment"
 	testTimeoutDuration      = time.Second * 2
 	testNamespace            = "testNamespace"
 	testInputError           = ""
 	testInputSuccessNumLines = 17
-	testInputSuccess         = `NAME                                 REPLICAS   READY    UPDATED   AVAILABLE   UNAVAILABLE
-	cdi-apiserver                        1          1        1         1           <none>
-	cdi-deployment                       1          1        1         1           <none>
-	cdi-operator                         1          1        1         1           <none>
-	cdi-uploadproxy                      1          1        1         1           <none>
-	cluster-network-addons-operator      1          1        1         1           <none>
-	hostpath-provisioner-operator        1          1        1         1           <none>
-	hyperconverged-cluster-operator      1          <none>   1         <none>      1
-	kubemacpool-mac-controller-manager   1          1        1         1           <none>
-	kubevirt-ssp-operator                1          <none>   1         <none>      1
-	nmstate-webhook                      2          2        2         2           <none>
-	node-maintenance-operator            1          <none>   1         <none>      1
-	v2v-vmware                           1          1        1         1           <none>
-	virt-api                             2          2        2         2           <none>
-	virt-controller                      2          2        2         2           <none>
-	virt-operator                        2          2        2         2           <none>
-	virt-template-validator              2          2        2         2           <none>
-	vm-import-operator                   0          <none>   <none>    <none>      <none>`
+	testInputSuccess         = `NAME                                 REPLICAS   READY    UPDATED   AVAILABLE   UNAVAILABLE  CURRENT 
```

Comment:
Need another input similar this but for stateful sets with some ready and some not ready